content/docs/next-release/deployment.md

```diff
@@ -472,6 +472,22 @@ usercert = ~/.cassandra/client-cert
 # validate = false
 ```
 
+### ScyllaDB
```

Comment:
Since ScyllaDB is not an officially supported backend, it's misleading to include it in this list.

We can add an L3 Section after Plugins called "Compatible Backends" and have a simple bullet list that includes ScyllaDB, with a link to the README in the main repo with Docker example.

Reply:
thanks, updated